test/objects.coffee

```diff
@@ -44,6 +44,15 @@ pluck = (x) -> x.a
 eq 100, pluck pluck pluck a: a: a: 100
 
 
+test "Object literal spread", ->
+  obj1 =
+    a: 1
+    b: 2
+  # obj2 = "#{{obj1..., b: 3}}"
+  t1 = CoffeeScript.compile "{obj1..., b: 3}", bare: yes
+  t1 = t1.replace(/\s+/g, "")
+  eq t1, "({...obj1,b:3});"
```

Comment:
Did you reply to the wrong comment? :)

Anyway, we need a test for the multiple splats error.